deploy/kubernetes/base/controller/psp.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: csi-gce-pd-controller-psp
+spec:
+  seLinux:
+    rule: RunAsAny
+  supplementalGroups:
+    rule: RunAsAny
+  runAsUser:
+    rule: RunAsAny
+  fsGroup:
+    rule: RunAsAny
+  volumes:
+    - "configMap"
```

Comment:
Why allow configMaps if the controller isn't currently using any?

Reply:
@mattcary good catch, no reason. I just copied our patch, removed.